util/cosmos/iscn/sign.js

```diff
@@ -204,7 +205,9 @@ export async function signISCNTx(
   const client = await getISCNSigningClient(signer);
   let res;
   if (iscnId) {
-    res = await client.updateISCNRecord(address, iscnId, payload, { memo, broadcast });
+    const iscnPrefix = getISCNPrefix(iscnId);
+    const { latestVersion } = await getISCNInfoById(iscnId);
```

Comment:
AShould use `iscnPrefix` for query instead

Reply:
updated